TRANSLATION.md

```diff
@@ -129,15 +130,18 @@ Terminy z (?) przy sugestii są do przegadania. Jeśli wiesz, jakie powinno ich
 | dispatcher | dyspozytor | Za pierwszym razem z dopiskiem (ang. *dispatcher*) |
 | dispatching an event | przesyłanie zdarzenia | |
 | dummy component | sztuczny komponent | |
+| effect event | efekt wydarzenia? wydarzenie efektu? | W kontekście `useEffectEvent` |
 | event handler | procedura obsługi zdarzenia | |
 | framework	| framework | |
 | function component | komponent funkcyjny | |
 | generic 'box' | "pojemnik" ogólnego użytku | Mowa o rodzaju komponentów, np. Dialog |
+| garbage collector | program czyszczący pamięć | |
 | glossary | słownik pojęć | |
 | hands-on tutorial	| praktyczny tutorial | |
 | higher-order components | komponenty wyższego rzędu | Za pierwszym razem z dopiskiem (ang. *higher-order components*) |
 | hook | hook | |
 | HTML entities | encje HTML | |
+| hydrate | hydratować? hydrować? | |
```

Comment:
Raczej "hydratuje". "Hydruje" powstałoby z "hydracji".
Co nie zmienia faktu, że to słowo jest tak dziwne i sztuczne, że nie mogę na nie patrzeć 😅
Z tego co kojarzę to tłumaczyłem to zwykle jakoś niedosłownie, np. "React odtworzy drzewo komponentów" zamiast "React zhydratuje drzewo komponentów".

Reply:
Hydratacja to proces chemiczny, w którym do jakiejś substancji dołączana jest woda i na koniec mamy uwodnioną substancję.

Zespół Reacta wymyślił sobie nazwę "hydration", który ma chyba tylko kojarzyć się i luźno nawiązywać do procesu chemicznego. W Reakcie statycznie wyrenderowane drzewo komponentów jest po stronie klienta uzupełniane reaktywnością i "mięskiem". Możliwe, że tutaj poszli po bandzie, sugerując się nazwą React, która nawiązuje do reagowania (też w chemii).

Można brnąć w to samo i zwyczajnie wstawiać poprawną odmianę czasownika "uwodnić", ale to już totalnie będzie się gryzło z programowaniem, więc może lepiej zostać z owianą nutką tajemniczości "hydratacją" i "hydratowaniem", mimo że to chamska kalka z angielskiego.